articles/cloudboost-cache.markdown

```diff
@@ -0,0 +1,143 @@
+Title: Turbo Charge your NodeJS app with Cache
+Author: Nawaz Dhandala
+Date: Tue Nov 24 2015 08:53:58 GMT-0700 (PDT)
+Node: v5.1.0
+
+
+Caching is great for your apps because it helps you to access data much faster when compared to the database. On the downside, querying is limited and it is very expensive (money-wise) because all the data is on the memory (which is expensive) instead of being on a disk. It is recommended that you use cache only for frequently accessed data.
+
+## Pre-Requisites
+
+### npm ###
+
+[npm](http://github.com/isaacs/npm) is the most popular package manager for Node.js.
+Installing npm is easy.
+
+    curl http://npmjs.org/install.sh | sh
+
+A note for windows users: npm does not currently work on windows. It will in the future.
+
```

Comment:
npm has come with node core for quite some time and works fine on windows using the official msi installer.